lib/Signaling/Listener.php

```diff
@@ -262,6 +263,9 @@ public static function notifyAfterRoomDisconected(ParticipantEvent $event): void
 	}
 
 	public static function notifyAfterJoinUpdateAndLeave(ModifyParticipantEvent $event): void {
+		if ($event instanceof SilentModifyParticipantEvent) {
```

Comment:
this is also not the correct listener...

Actually what you want to opt out is this one:
https://github.com/nextcloud/spreed/blob/master/lib/Notification/Listener.php#L97-L101

Reply:
Every when we create a room, this event is called and define the attribute room.last_activity with the current date.

https://github.com/nextcloud/spreed/blob/master/lib/Chat/SystemMessage/Listener.php#L83

The event:
https://github.com/nextcloud/spreed/blob/master/lib/Notification/Listener.php#L97-L101
Check this:
https://github.com/nextcloud/spreed/blob/master/lib/Notification/Listener.php#L195
And if the last_activity have a date, will never send the call notification.